meta_review/data.py

```diff
@@ -0,0 +1,657 @@
+            participants.append(participant)
+        try:
+            # there's no way to do bulk update, so we delete and then create
+            # bulk delete all participants
+            Participant.objects.all().delete()
+            # bulk create all participants
+            Participant.objects.bulk_create(participants)
+        except Exception as ex:
+            self.logger.error(
+                'Something went wrong saving participants: %s' % ex)
+
+    def __dump_comments_to_database(self):
+        """
+        Dump comments data into Django database
+        """
+        self.logger.info('dump review comments data into database')
+        comments = []
+        for key, comment in self.comments.items():
+            comments.append(comment)
+        try:
+            # there's no way to do bulk update, so we delete and then create
+            # bulk delete all comments
+            Comment.objects.all().delete()
+            # bulk create all comments
+            Comment.objects.bulk_create(comments)
+        except Exception as ex:
+            self.logger.error(
+                'Something went wrong saving comments: %s' % ex)
+
+    def __dump_reactions_to_database(self):
+        """
+        Dump reactions data into Django database
+        """
+        self.logger.info('dump reactions data into database')
+        reactions = []
+        for key, reaction in self.reactions.items():
+            reactions.append(reaction)
+        try:
+            # there's no way to do bulk update, so we delete and then create
+            # bulk delete all reactions
+            Reaction.objects.all().delete()
+            # bulk create all reactions
+            Reaction.objects.bulk_create(reactions)
+        except Exception as ex:
+            self.logger.error(
+                'Something went wrong saving reactions: %s' % ex)
+
+
+def handle():
+    # load data from gh-board repo
+    org_name = get_org_name()
+
+    # URL to grab all issues from
+    issues_url = 'http://' + org_name + '.github.io/gh-board/issues.json'
+
+    logger = logging.getLogger(__name__)
+
+    try:
+        content = requests.get(issues_url, timeout=10)
+        content.raise_for_status()
+    except Exception as ex:
+        logger.warning('Get issues from %s failed: %s Try backup url'
+                       % (issues_url, ex))
+        issues_url = 'https://' + org_name + '-gh-board.netlify.com/issues.json'
+        try:
+            content = requests.get(issues_url, timeout=10)
+            content.raise_for_status()
+        except Exception as ex:
+            logger.error('Get issues from %s failed: %s'
+                         % (issues_url, ex))
+
+    try:
+        parsed_json = content.json()
+        handler = MetaReviewHandler(parsed_json['issues'], timezone.now())
+        handler.handle()
+    except json.JSONDecodeError as ex:
+        logger.error('JSON decode error: %s' % ex)
```

Comment:
Consider breaking this down to three files, one for handlers, one for all the update functions and one for all the loader functions ?